### YamlMime:ManagedReference
items:
- uid: System.Security.XmlSyntaxException
  id: XmlSyntaxException
  children:
  - System.Security.XmlSyntaxException.#ctor
  - System.Security.XmlSyntaxException.#ctor(System.Int32)
  - System.Security.XmlSyntaxException.#ctor(System.String)
  - System.Security.XmlSyntaxException.#ctor(System.Int32,System.String)
  - System.Security.XmlSyntaxException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: XmlSyntaxException
  nameWithType: XmlSyntaxException
  fullName: System.Security.XmlSyntaxException
  type: Class
  summary: "A exceção que é lançada quando há um erro de sintaxe ao analisar o XML. Essa classe não pode ser herdada."
  remarks: "XmlSyntaxException usa o padrão <xref:System.Object.Equals%2A>implementação, que oferece suporte a igualdade de referência.</xref:System.Object.Equals%2A>       Para obter uma lista de valores de propriedade inicial para uma instância de XmlSyntaxException, consulte o construtor."
  syntax:
    content: >-
      [System.Runtime.InteropServices.ComVisible(true)]

      public sealed class XmlSyntaxException : SystemException
  inheritance:
  - System.Exception
  - System.SystemException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.Security.XmlSyntaxException.#ctor
  id: '#ctor'
  parent: System.Security.XmlSyntaxException
  langs:
  - csharp
  name: XmlSyntaxException()
  nameWithType: XmlSyntaxException.XmlSyntaxException()
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException()
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Security
  summary: "Inicializa uma nova instância do <xref href=&quot;System.Security.XmlSyntaxException&quot;> </xref> classe com propriedades padrão."
  remarks: "A tabela a seguir mostra os valores de propriedade inicial para uma instância de <xref:System.Security.XmlSyntaxException>.</xref:System.Security.XmlSyntaxException>      | Propriedade | Valor |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>|&quot; Sintaxe inválida. &quot;|</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public XmlSyntaxException ();
    parameters: []
  overload: System.Security.XmlSyntaxException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Security.XmlSyntaxException.#ctor(System.Int32)
  id: '#ctor(System.Int32)'
  parent: System.Security.XmlSyntaxException
  langs:
  - csharp
  name: XmlSyntaxException(Int32)
  nameWithType: XmlSyntaxException.XmlSyntaxException(Int32)
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException(Int32)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Security
  summary: "Inicializa uma nova instância do <xref href=&quot;System.Security.XmlSyntaxException&quot;> </xref> classe com o número da linha em que a exceção foi detectada."
  remarks: "O número de linha de um fluxo XML é atualizado sempre que um caractere de nova linha é detectado. Numeração começa com 1 para a primeira linha.       A tabela a seguir mostra os valores de propriedade inicial para uma instância de <xref:System.Security.XmlSyntaxException>.</xref:System.Security.XmlSyntaxException>      | Propriedade | Valor |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>|&quot; Sintaxe inválida na linha `lineNumber`. &quot;|</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public XmlSyntaxException (int lineNumber);
    parameters:
    - id: lineNumber
      type: System.Int32
      description: "O número da linha do fluxo XML onde o erro de sintaxe XML foi detectado."
  overload: System.Security.XmlSyntaxException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Security.XmlSyntaxException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Security.XmlSyntaxException
  langs:
  - csharp
  name: XmlSyntaxException(String)
  nameWithType: XmlSyntaxException.XmlSyntaxException(String)
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException(String)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Security
  summary: "Inicializa uma nova instância do <xref href=&quot;System.Security.XmlSyntaxException&quot;> </xref> classe com uma mensagem de erro especificada."
  remarks: "A tabela a seguir mostra os valores de propriedade inicial para uma instância de <xref:System.Security.XmlSyntaxException>.</xref:System.Security.XmlSyntaxException>      | Propriedade | Valor |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| A cadeia de caracteres de mensagem de erro. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public XmlSyntaxException (string message);
    parameters:
    - id: message
      type: System.String
      description: "A mensagem de erro que explica o motivo da exceção."
  overload: System.Security.XmlSyntaxException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Security.XmlSyntaxException.#ctor(System.Int32,System.String)
  id: '#ctor(System.Int32,System.String)'
  parent: System.Security.XmlSyntaxException
  langs:
  - csharp
  name: XmlSyntaxException(Int32,String)
  nameWithType: XmlSyntaxException.XmlSyntaxException(Int32,String)
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException(Int32,String)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Security
  summary: "Inicializa uma nova instância do <xref href=&quot;System.Security.XmlSyntaxException&quot;> </xref> classe com uma mensagem de erro especificada e o número da linha em que a exceção foi detectada."
  remarks: "O número de linha de um fluxo XML é atualizado sempre que um caractere de nova linha é detectado. Numeração começa com 1 para a primeira linha.       A tabela a seguir mostra os valores de propriedade inicial para uma instância de <xref:System.Security.XmlSyntaxException>.</xref:System.Security.XmlSyntaxException>      | Propriedade | Valor |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>|&quot; Sintaxe inválida na linha `lineNumber`  -  `message` &quot;|</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public XmlSyntaxException (int lineNumber, string message);
    parameters:
    - id: lineNumber
      type: System.Int32
      description: "O número da linha do fluxo XML onde o erro de sintaxe XML foi detectado."
    - id: message
      type: System.String
      description: "A mensagem de erro que explica o motivo da exceção."
  overload: System.Security.XmlSyntaxException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Security.XmlSyntaxException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Security.XmlSyntaxException
  langs:
  - csharp
  name: XmlSyntaxException(String,Exception)
  nameWithType: XmlSyntaxException.XmlSyntaxException(String,Exception)
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException(String,Exception)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Security
  summary: "Inicializa uma nova instância do <xref href=&quot;System.Security.XmlSyntaxException&quot;> </xref> classe com uma mensagem de erro especificada e uma referência à exceção interna que é a causa dessa exceção."
  remarks: "Uma exceção que é lançada como um resultado direto de uma exceção anterior deve incluir uma referência à exceção anterior no <xref:System.Exception.InnerException%2A>propriedade.</xref:System.Exception.InnerException%2A> O <xref:System.Exception.InnerException%2A>propriedade retorna o mesmo valor que é passado para o construtor, ou `null` se o <xref:System.Exception.InnerException%2A>propriedade não fornecer o valor da exceção interna para o construtor.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       A tabela a seguir mostra os valores de propriedade inicial para uma instância de <xref:System.Security.XmlSyntaxException>.</xref:System.Security.XmlSyntaxException>      | Propriedade | Valor |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| A referência à exceção interna. |   | <xref:System.Exception.Message%2A>| A cadeia de caracteres de mensagem de erro. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public XmlSyntaxException (string message, Exception inner);
    parameters:
    - id: message
      type: System.String
      description: "A mensagem de erro que explica o motivo da exceção."
    - id: inner
      type: System.Exception
      description: "A exceção que é a causa da exceção atual. Se o `inner` parâmetro não é <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>, a exceção atual é gerada em um <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> bloco que manipula a exceção interna."
  overload: System.Security.XmlSyntaxException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.SystemException
  isExternal: false
  name: System.SystemException
- uid: System.Security.XmlSyntaxException.#ctor
  parent: System.Security.XmlSyntaxException
  isExternal: false
  name: XmlSyntaxException()
  nameWithType: XmlSyntaxException.XmlSyntaxException()
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException()
- uid: System.Security.XmlSyntaxException.#ctor(System.Int32)
  parent: System.Security.XmlSyntaxException
  isExternal: false
  name: XmlSyntaxException(Int32)
  nameWithType: XmlSyntaxException.XmlSyntaxException(Int32)
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException(Int32)
- uid: System.Int32
  parent: System
  isExternal: true
  name: Int32
  nameWithType: Int32
  fullName: System.Int32
- uid: System.Security.XmlSyntaxException.#ctor(System.String)
  parent: System.Security.XmlSyntaxException
  isExternal: false
  name: XmlSyntaxException(String)
  nameWithType: XmlSyntaxException.XmlSyntaxException(String)
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Security.XmlSyntaxException.#ctor(System.Int32,System.String)
  parent: System.Security.XmlSyntaxException
  isExternal: false
  name: XmlSyntaxException(Int32,String)
  nameWithType: XmlSyntaxException.XmlSyntaxException(Int32,String)
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException(Int32,String)
- uid: System.Security.XmlSyntaxException.#ctor(System.String,System.Exception)
  parent: System.Security.XmlSyntaxException
  isExternal: false
  name: XmlSyntaxException(String,Exception)
  nameWithType: XmlSyntaxException.XmlSyntaxException(String,Exception)
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Security.XmlSyntaxException.#ctor*
  parent: System.Security.XmlSyntaxException
  isExternal: false
  name: XmlSyntaxException
  nameWithType: XmlSyntaxException.XmlSyntaxException
